packages/common/src/services/audius-backend/solana.ts

```diff
@@ -782,11 +782,11 @@ export const recoverUsdcFromRootWallet = async ({
       ],
       mint: 'USDC'
     })
-  const transaction = await sdk.services.claimableTokensClient.buildTransaction(
-    { instructions: [memoInstruction, transferInstruction, routeInstruction] }
-  )
+  const transaction = await sdk.services.solanaClient.buildTransaction({
+    instructions: [memoInstruction, transferInstruction, routeInstruction]
+  })
   transaction.sign([sender])
-  const signature = await sdk.services.paymentRouterClient.sendTransaction(
+  const signature = await sdk.services.claimableTokensClient.sendTransaction(
```

Comment:
wait why is this one not `solanaClient`?

Reply:
Oh yeah. It should be! Good catch